SilKit/source/core/internal/traits/SilKitMsgTraits.hpp

```diff
@@ -14,6 +14,11 @@ namespace Core {
 // ==================================================================
 //  Trait which checks that '.timestamp' works
 // ==================================================================
+template<class T>
+struct RemoveCvRef
+{
+    using type = std::remove_cv_t<std::remove_reference_t<T>>;
+};
```

Comment:
You could switch to a templated typedef, then you don't have to use the `typename RemoveCvRef<...>::type` and can just write `RemoveCvRef<...>`.

```suggestion
template<class T>
using RemoveCvRef = std::remove_cv_t<std::remove_reference_t<T>>;
```